### PR TITLE
[PT-BR] Add content/pt-br/docs/tasks/configmap-secret/

### PR DESCRIPTION
**Pull request by edsoncelio**
## Changes
* Translation to content/pt-br/docs/tasks/configmap-secret/ in pt-br

### CODE REVIEW

File: content/pt-br/docs/tasks/configmap-secret/_index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: "Gerenciando Secrets"
+weight: 28
+description: Gerenciando dados de configurações confidencias usando Secrets.
```
Comment (rikatz) [inline content/pt-br/docs/tasks/configmap-secret/_index.md:4]: Although in original English it's using the "confidential" word, I'm a bit afraid that this wording makes users things that secrets and configmaps are secure and encrypted (which they are not, unless you configure them and just to be encrypted at rest).

Just some though.

Also, typo in confidencias / confidenciais

File: content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+<!-- steps -->
+
+## Crie o arquivo de configuração
+
+Você pode criar um Secret primeiramente em um arquivo, no formato JSON ou YAML, e depois
```
Comment (rikatz) [inline content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md:18]: do we have a definition of um secret or uma secret? I'm ok following with "um secret" (as the secret translation is "segredo" which is a masculine word), but just want to check the consistency over the other docs

File: content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+criar o objeto. O recurso [Secret](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#secret-v1-core)
+contém dois *maps*: `data` e `stringData`.
```
Comment (rikatz) [inline content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md:20]: maybe maps should be in italic (_maps_) if we decide not to translate it. But I guess here it makes some sense to have "mapas" as this is a common development word translated to portuguese

File: content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+echo -n '1f2d1e2e67df' | base64
+```
+
+A saída deve ser similar a:
+
+```
+MWYyZDFlMmU2N2Rm
+```
+
+Escreva o arquivo de configuração do Secret, que ser parecido com:
```
Comment (rikatz) [inline content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md:49]: que será parecido com

File: content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+Perceba que o nome do objeto Secret precisa ser um 
+[nome de subdomínio DNS](/docs/concepts/overview/working-with-objects/names#dns-subdomain-name) válido.
+
+{{< note >}}
+Os valores serializados dos dados JSON e YAML de um Secret são codificados em strings
+base64. Novas linhas não são válidas com essas strings e devem ser omitidas. Quando
+usar o utilitário `base64` em Darwin/MacOS, os usuários devem evitar usar a opção `-b`
+para separar linhas grandes. Por outro lado, usuários de Linux *devem* adicionar a opção
+`-w 0` ao comando `base64` ou o *pipe* `base64 | tr -d '\n'` se a opção `w` não for disponível
```
Comment (rikatz) [inline content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md:69]: suggestion: se a opção w não estiver disponível

File: content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+`-w 0` ao comando `base64` ou o *pipe* `base64 | tr -d '\n'` se a opção `w` não for disponível 
+{{< /note >}}
+
+Para cenários específicos, você pode querer usar o campo `stringData` ao invés de `data`.
+Esse campo permite que você use strings não-base64 diretamente dentro do Secret, 
+e a string vai ser codificada para você quando o Secret for criado ou atualizado.
+
+Um exemplo prático para isso pode ser quando você esteja fazendo *deploy* de uma aplicação
+que usa um Secret para armazenar um arquivo de configuração, e você quer popular partes desse
+arquivo de configuração durante o processo de *deployment*.
```
Comment (rikatz) [inline content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md:78]: processo de implantação maybe?

File: content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+  config.yaml: YXBpVXJsOiAiaHR0cHM6Ly9teS5hcGkuY29tL2FwaS92MSIKdXNlcm5hbWU6IHt7dXNlcm5hbWV9fQpwYXNzd29yZDoge3twYXNzd29yZH19
+```
+
+Os comandos `kubectl get` e `kubectl describe` omitem o conteúdo de um `Secret` por padrão.
+Isso para proteger o `Secret` de ser exposto acidentalmente para uma pessoa não autorizada,
+ou ser armazenado em um log de terminal.
+Para verificar o conteúdo atual de um dado codificado, veja [decodificando secret](/docs/tasks/configmap-secret/managing-secret-using-kubectl/#decoding-secret).
+
+Se um campo, como `username`, é especificado em `data` e `stringData`,
+o valor de `stringData` é o usado. Por exemplo, dado a seguinte definição do Secret:
```
Comment (rikatz) [inline content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md:148]: dado or dada a seguinte definição? not sure :)

File: content/pt-br/docs/tasks/configmap-secret/managing-secret-using-kustomize.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+weight: 30
+description: Criando objetos Secret usando o arquivo kustomization.yaml
+---
+
+<!-- overview -->
+
+Desde o Kubernetes v1.14, o `kubectl` provê suporte para [gerenciamento de objetos usando Kustomize](/docs/tasks/manage-kubernetes-objects/kustomization/).
+O Kustomize provê geradores de recursos para criar Secrets e ConfigMaps. 
+Os geradores Kustomize devem ser especificados em um arquivo `kustomization.yaml` dentro
+de um diretório. Depois de gerar o Secret, você pode criar o Secret na API server com `kubectl apply`.
```
Comment (rikatz) [inline content/pt-br/docs/tasks/configmap-secret/managing-secret-using-kustomize.md:13]: maybe we can have some translation not pointing to the API Server (it became confusing here), but something like "você pode criar o Secret com" and skip the API Server

File: content/pt-br/docs/tasks/configmap-secret/managing-secret-using-kustomize.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+
+```yaml
+secretGenerator:
+- name: db-user-pass
+  files:
+  - username.txt
+  - password.txt
+```
+
+Você também pode definir o `secretGenerator`no arquivo `kustomization.yaml`
```
Comment (rikatz) [inline content/pt-br/docs/tasks/configmap-secret/managing-secret-using-kustomize.md:34]: need space here between `secretGenerator` and no

**Review by rikatz — CHANGES_REQUESTED** [review]
Thanks for the contribution.

Left some minor reviews here :)

File: content/pt-br/docs/tasks/configmap-secret/_index.md (hunk 1 above)
Comment (edsoncelio) [reply content/pt-br/docs/tasks/configmap-secret/_index.md:4]: I agree with that, maybe we can remove the 'Confidenciais' word and just use 'dados de configurações usando Secrets', what you think?

File: content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md (hunk 2 above)
Comment (edsoncelio) [reply content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md:18]: Just to let you know, I found some others use of 'um secret' at the reference docs in pt-br, so I guess it's okay continue with this translation.

File: content/pt-br/docs/tasks/configmap-secret/_index.md (hunk 1 above)
Comment (jcjesus) [reply content/pt-br/docs/tasks/configmap-secret/_index.md]: @edsoncelio Use "Credenciais" instead "Secrets", what do you think?

Comment (edsoncelio) [reply content/pt-br/docs/tasks/configmap-secret/_index.md]: Hey @jcjesus thanks for the suggestion!  just to know, you're talking about the word 'Secrets' in the title or description?

Comment (jcjesus) [reply content/pt-br/docs/tasks/configmap-secret/_index.md]: I think you can use both items.

Comment (edsoncelio) [reply content/pt-br/docs/tasks/configmap-secret/_index.md]: Nice, I agree with the first one, but to the second, the word 'Secret' it's used as a kubernetes resource name that will be use to create the credentials...

Comment (jcjesus) [reply content/pt-br/docs/tasks/configmap-secret/_index.md]: Sure, I agree too! Thanks!

**Comment by rikatz** [comment]
/lgtm
/approve
Thanks!!

**Pull request merged**